chef-config/lib/chef-config/fips.rb

```diff
@@ -16,10 +16,12 @@
 # limitations under the License.
 #
 
+require "chef-utils"
```

Comment:
Do you want to gate this here similar to above?

Reply:
yeah there's a bunch of gates i still need to fix